master/static/migrations/20230718150024_trial-one-to-many-tasks.tx.up.sql

```diff
@@ -0,0 +1,76 @@
+CREATE TABLE public.trial_id_task_id (
+    trial_id integer REFERENCES trials(id) ON DELETE CASCADE NOT NULL,
+    task_id text REFERENCES tasks(task_id) ON DELETE CASCADE NOT NULL UNIQUE,
+    task_run_id integer DEFAULT 0 NOT NULL, -- Starts at 0. Trial gets restarted goes to 1.
+    PRIMARY KEY(trial_id, task_id)
+);
+
+CREATE INDEX idx_trial_id_task_id ON public.trial_id_task_id USING btree (trial_id);
+
+CREATE INDEX idx_trial_id_task_id_task_id ON public.trial_id_task_id USING btree (task_id);
+
+CREATE INDEX idx_trial_id_task_id_run_id ON public.trial_id_task_id USING btree (trial_id, task_run_id);
```

Comment:
if we get rid of task_run_id, i guess we would need to create an index on tasks.start_time

Reply:
Added index for tasks.start_time.